README.md

```diff
@@ -63,11 +63,11 @@ Developers choose Citus for two reasons:
 
 ## Getting Started
 
-The quickest way to get started with Citus is to use the [Hyperscale (Citus)](https://docs.microsoft.com/azure/postgresql/quickstart-create-hyperscale-portal) deployment option in the Azure Database for PostgreSQL managed service—or [set up Citus locally](https://docs.citusdata.com/en/stable/installation/single_node.html).
+The quickest way to get started with Citus is to use the [Azure Cosmos DB for PostgreSQL](https://learn.microsoft.com/en-us/azure/cosmos-db/postgresql/quickstart-create-portal) deployment option in the Azure Cosmos DB for PostgreSQL managed service—or [set up Citus locally](https://docs.citusdata.com/en/stable/installation/single_node.html).
 
-### Hyperscale (Citus) on Azure Database for PostgreSQL
+### Azure Cosmos DB for PostgreSQL
 
-You can get a fully-managed Citus cluster in minutes through the Hyperscale (Citus) deployment option in the [Azure Database for PostgreSQL](https://azure.microsoft.com/services/postgresql/) portal. Azure will manage your backups, high availability through auto-failover, software updates, monitoring, and more for all of your servers. To get started with Hyperscale (Citus), use the [Hyperscale (Citus) Quickstart](https://docs.microsoft.com/azure/postgresql/quickstart-create-hyperscale-portal) in the Azure docs.
+You can get a fully-managed Citus cluster in minutes through the Azure Cosmos DB for PostgreSQL deployment option in the [Azure Cosmos DB for PostgreSQL](https://azure.microsoft.com/en-us/products/cosmos-db/) portal. Azure will manage your backups, high availability through auto-failover, software updates, monitoring, and more for all of your servers. To get started with Azure Cosmos DB for PostgreSQL, use the [Azure Cosmos DB for PostgreSQL Quickstart](https://learn.microsoft.com/en-us/azure/cosmos-db/postgresql/quickstart-create-portal) in the Azure docs.
```

Comment:
1) Please remove "/en-us" from the docs/learn URL, so that the URL will redirect to whichever locale is set on the reader's system
2) Need to remove the "deployment option" term too
3) I think the tricky part is the name of our managed service used to have 2 parts: Hyperscale (Citus) and Azure Database for PostgreSQL. Above, we've replaced BOTH parts with "Azure Cosmos DB for PostgreSQL" which is creating a repetition. Can you please remove the repetition, in both the 1st sentence on this line, and in the 3rd sentence too!